src/components/PostDetails/PostDetails.css

```diff
@@ -1,22 +1,17 @@
 .post-details-section {
```

Comment:
As a usuer - I'm dying for a home button or back button of some sort.

Reply:
The whole logo/h1 is a link now, do you also want home?